src/Symfony/Component/Security/Core/Resources/translations/security.hy.xlf

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>An authentication exception occurred.</source>
+                <target>Նույնականացման սխալ։</target>
+            </trans-unit>
+            <trans-unit id="2">
+                <source>Authentication credentials could not be found.</source>
+                <target>Նույնականացման տվյալները չեն գտնվել։</target>
+            </trans-unit>
+            <trans-unit id="3">
+                <source>Authentication request could not be processed due to a system problem.</source>
+                <source>Համակարգային սխալ՝ նույնականացման հացրման պրոցեսինգի ժամանակ։</source>
+                <target></target>
```

Comment:
Oops. I made a misstake that broke Travis. This change will fix it. 

```suggestion
                <target>Համակարգային սխալ՝ նույնականացման հացրման պրոցեսինգի ժամանակ։</target>
```

Reply:
@Nyholm Please fix this and send me true suggestion. I think we are already fixed this

Reply:
My first fix was wrong. I introduced 2 "source" elements. I added this change and now travis is green.